client/package.json

```diff
@@ -244,6 +243,7 @@
 		"react-router": "^6.23.1",
 		"react-test-renderer": "^18.3.1",
 		"redux-mock-store": "^1.5.5",
+		"storybook": "^8.6.9",
```

Comment:
We now need this as an explicit dependency.

(Same with the other package.jsons)